src/format/KeePass2.cpp

```diff
@@ -133,3 +129,29 @@ KeePass2::ProtectedStreamAlgo KeePass2::idToProtectedStreamAlgo(quint32 id)
         return KeePass2::ProtectedStreamAlgo::InvalidProtectedStreamAlgo;
     }
 }
+
+QString KeePass2::cipherToString(QUuid cipherUuid)
+{
+    if (cipherUuid == KeePass2::CIPHER_AES256) {
```

Comment:
What about `CIPHER_AES128` which is defined on line 25? I know it was not mentioned in the original `CIPHERS` list but if it is still used it will be translated into the `Invalid Cipher`.

Reply:
This cipher is only used with KeePass1 databases and I think the AES KDF, the string isn't exposed anywhere.

Reply:
Got it.